ld.script

```diff
@@ -0,0 +1,233 @@
+/* Script for -z combreloc: combine and sort reloc sections */
+/* Copyright (C) 2014-2015 Free Software Foundation, Inc.
+   Copying and distribution of this script, with or without modification,
+   are permitted in any medium without royalty provided the copyright
+   notice and this notice are preserved.  */
```

Comment:
too low level for me to review, how do we make sure its correctness? are there documentations around this?

Reply:
There were 3 line modification to the default ld.script to create a variable name(s) for the start of actual node text segment.
>   PROVIDE (__nodetext = .);
>   PROVIDE (_nodetext = .);
>   PROVIDE (nodetext = .);